src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/pim_configurations/pim_osl_configuration.py

```diff
@@ -0,0 +1,23 @@
+# ©2023, ANSYS Inc. Unauthorized use, distribution or duplication is prohibited.
+
+import sys
+
+from pathlib import Path
+from typing import List
+
+from ansys.saf.glow.desktop.orchestration.pim_configurations import PimProductConfigurationBuilder
+from ansys.saf.glow.desktop.orchestration.pim_process import PimProductConfiguration
+
+
+class OptislangConfigBuilder(PimProductConfigurationBuilder):
+    def get_configurations(self) -> List[PimProductConfiguration]:
+        uvicorn_exe = Path(sys.executable).parent / "uvicorn"
+        return [
+            PimProductConfiguration(
+                "optislang",
+                str(uvicorn_exe),
+                arguments=["ansys.solutions.{{ cookiecutter.__solution_name_slug }}.pim_configurations.osl_wrapper:app", "--port", "${AENEID_PORT_HTTP}"],
```

Comment:
@iazehaf I am not sure I understand, in some parts, the solution is hardcoded to be "calculator" and in some other places, it is still templated?

Reply:
This is a mistake. Solved.